processing/src/main/java/org/apache/carbondata/processing/sort/sortdata/SortParameters.java

```diff
@@ -532,7 +556,7 @@ public static SortParameters createSortParameters(CarbonDataLoadConfiguration co
       parameters.setNoDictSortDataType(noDictSortAndNoSortDataTypes.get("noDictSortDataTypes"));
       parameters.setNoDictNoSortDataType(noDictSortAndNoSortDataTypes.get("noDictNoSortDataTypes"));
       // keep partition columns in the end for table spec by getting rearranged tale spec
-      TableSpec tableSpec = new TableSpec(configuration.getTableSpec().getCarbonTable(), true);
+      TableSpec tableSpec = new TableSpec(configuration.getTableSpec().getCarbonTable(), false);
```

Comment:
a) Inside TableSpec, If the true part is dead code, please remove it.
b) remove the above comments, it's not applicable anymore.
c) Please check all the test case with `carbon.enable.bad.record.handling.for.insert` true also.
  partition column, same sort column (all the type, measure & dim) [ local sort, global, no sort]
  partition column without sort columns
  partition with other sort columns (all the type, measure & dim) [ local sort, global, no sort]

Reply:
Some more scenario is that we can check like,
iv) more than one partition column (one int, one string) and more than one sort column for above scenarios.
v) static partition and dynamic partition together

Reply:
Rework is done and tested above-mentioned scenario. It is working fine now. @ajantha-bhat please review.

Reply:
@nihal0107 i dont see test cases for many of the scenarios mentioned.

Reply:
For some of the scenario test cases were already present like:
a) partition column without sort columns
b) partition with other sort columns (all the type, measure & dim) [ local sort, global, no sort]
c) static partition and dynamic partition together

and test case which I have added covered for some of flow like:
a) partition column, same sort column (all the type, measure & dim) [ local sort, global, no sort]
b) more than one partition column (one int, one string) and more than one sort column for above scenarios.

Reply:
ok